ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js

```diff
@@ -21,9 +21,104 @@
         static NS = FormView.Constants.NS;
         static IS = "adaptiveFormContainer";
         static bemBlock = 'cmp-adaptiveform-container';
+        static hamburgerSupport = false;
+        static nestingSupport = 3;
+
         static selectors  = {
             self: "[data-" + this.NS + '-is="' + this.IS + '"]',
+            menu: `.${FormContainerV2.bemBlock}-menu`,
+            hamburgerMenu: `.${FormContainerV2.bemBlock}-hamburgerMenu`,
+            hamburgerMenuIcon: `.${FormContainerV2.bemBlock}-hamburger-icon`,
+            hamburgerSubMenu: `.${FormContainerV2.bemBlock}-submenu`,
+            hamburgerMenuContainer: `.${FormContainerV2.bemBlock}-hamburgerMenuContainer`,
+            topContainer: `.${FormContainerV2.bemBlock}-top-container`,
+            containerMenu: `.${FormContainerV2.bemBlock}-container-menu`,
+            navBar: `.${FormContainerV2.bemBlock}-nav-bar`,
+            navTitle: `.${FormContainerV2.bemBlock}-nav-title`,
+            navLink: `.${FormContainerV2.bemBlock}-nav-link`,
+            upNavButton: `.${FormContainerV2.bemBlock}-nav-button-up`,
```

Comment:
Can we rename the variables as follows?
upNavButton to closeSubMenu
downNavButton to openSubMenu
leftNavButton to previousButton
rightNavButton to nextButton

Reply:
Done